core/container.go

```diff
@@ -101,24 +67,8 @@ func ContainerManager() string {
 	return ""
 }
 
-func GetHostImage() (img string, err error) {
-	if settings.Cnf.Image != "" {
-		return settings.Cnf.Image, nil
-	}
-
-	distro_raw, err := exec.Command("lsb_release", "-is").Output()
-	if err != nil {
-		return "", err
-	}
-	distro := strings.ToLower(strings.Trim(string(distro_raw), "\r\n"))
-
-	release_raw, err := exec.Command("lsb_release", "-rs").Output()
-	if err != nil {
-		return "", err
-	}
-	release := strings.ToLower(strings.Trim(string(release_raw), "\r\n"))
-
-	return fmt.Sprintf("%v:%v", distro, release), nil
+func GetContainerName(distro settings.DistroInfo) (name string) {
+	return "apx_managed_" + distro.Id
```

Comment:
We can probably remove this function, since the default names are already stored in the structs